src/main/java/io/codelex/enums/Exercise2/RockPaperScissors.java

```diff
@@ -0,0 +1,71 @@
+package io.codelex.enums.Exercise2;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RockPaperScissors {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int numOfPlays = 1;
+        int wins = 0;
+        int losses = 0;
+
+        System.out.println("Let us begin...");
+
+        while (true) {
+
+            System.out.println("Scissor-Paper-Stone");
+            System.out.println("Your turn (Enter 'S' for scissors, 'P' for paper, 'R' for rock, q to quit): ");
+
+            String playerMove = scanner.nextLine().toLowerCase();
+            
+
+            if (playerMove.equals("quit") || playerMove.equals("q")) {
+                System.out.println("Number of trials: " + numOfPlays);
+                System.out.println("I won " + losses + "(" + ((float) (losses * 100 / numOfPlays))
+                        + "%). You won " + wins + "(" + ((float) (wins * 100 / numOfPlays)) + "%).");
+                System.out.println("Bye!");
+                break;
+            }
+
+            if (Arrays.stream(Move.values()).noneMatch(x -> x.getMove().equals(playerMove))) {
```

Comment:
noteikti par garu priekš if statemnt, uz palīgmetodi
un x varētu smukāk ar kaut kādu `item` vai ko tādu, pats izmantoju `it` vienmēr

Reply:
Salabots.